readme.md

```diff
@@ -93,6 +93,13 @@ prompt pure
 | **`PURE_GIT_DOWN_ARROW`**        | Defines the git down arrow symbol.                                                             | `⇣`            |
 | **`PURE_GIT_UP_ARROW`**          | Defines the git up arrow symbol.                                                               | `⇡`            |
 
+## Colors
+
+Colors can be changed by using `zstyle` with the pattern
+`:prompt:pure:$color_name` and style `color`. The availaible color names are:
+`branch`, `error`, `exec_time`, `git_arrow`, `host`, `path`, `root`, `success`,
+`user` and `virtualenv`.
```

Comment:
I think the docs here needs to be expanded. It should be clearer what the different parts actually are. I would make it a list and add a screenshot showing each component with an arrow.

Also don't hard-wrap.

Reply:
Thank you for taking the time to review this PR.

I have removed the hard-wraps and extend the documentation to explain what parts of the prompt each color affect. However I didn't made a visual explanation as you requested because GUI application are not my *forte* and I don't see how to annotate a screenshot with ten colors without overcrowding it. But if you insist I'll try.

Reply:
I'm happy to consider other ways to document it, but we need to make it perfectly clear what the different parts are.

Reply:
I have added a ASCII diagram that shows exactly which parts are colored by which colors. WDYT?

Reply:
Looks great!